Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```
@@ -564,16 +564,29 @@
     - Kangaroo
 
 - type: entity
-  parent: ClothingBeltStorageBase
+  parent: ClothingNeckBase
   id: ClothingBeltHolster
   name: shoulder holster
   description: 'A holster to carry a handgun and ammo. WARNING: Badasses only.'
   components:
+  - type: UserInterface
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)